MODULE.bazel

```diff
@@ -8,9 +8,10 @@ module(
 
 # Lower-bounds (minimum) versions for direct runtime dependencies
 bazel_dep(name = "aspect_bazel_lib", version = "1.30.2")
-bazel_dep(name = "aspect_rules_js", version = "1.29.2")
+bazel_dep(name = "aspect_rules_js", version = "1.31.0")
 bazel_dep(name = "bazel_skylib", version = "1.4.1")
 
 ####### Dev dependencies ########
 
+bazel_dep(name = "bazel_features", version = "0.1.0", dev_dependency = True)
```

Comment:
Is this actually bazel7 related?

Reply:
indirectly... the minimum rules_js version that works with Bazel 7 is 1.31.0 and that version has this odd requirements for bazel_features that snuck in unintentionally.... the requirement is dropped in the latest rules_js release so it will get dropped on the bump to rules_js 2